lib/matplotlib/ticker.py

```diff
@@ -688,29 +777,38 @@ def _formatSciNotation(self, s):
 
 class LogFormatter(Formatter):
     """
-    Format values for log axis;
-
+    Format values for log axis.
     """
     def __init__(self, base=10.0, labelOnlyBase=True):
         """
-        *base* is used to locate the decade tick,
-        which will be the only one to be labeled if *labelOnlyBase*
-        is ``False``
+        `base` is used to locate the decade tick, which will be the only
+        one to be labeled if `labelOnlyBase` is ``True``.
         """
         self._base = base + 0.0
         self.labelOnlyBase = labelOnlyBase
 
     def base(self, base):
-        """change the *base* for labeling - warning: should always match the
-           base used for :class:`LogLocator`"""
+        """
+        change the `base` for labeling.
+
+        .. warning::
+           Should always match the base used for :class:`LogLocator`
+
+        """
         self._base = base
 
     def label_minor(self, labelOnlyBase):
-        'switch on/off minor ticks labeling'
+        """
+        Switch minor tick labeling on or off.
+
+        ``labelOnlyBase=True`` to turn off minor ticks.
+        """
```

Comment:
This method does nothing for this particular class since `self.labelOnlyBase` is not used in `self.pprint_val`. However, it does work for the `LogFormatterExponent` class, which extends this one. The old docs and the method name are indeed very misleading. `labelOnlyBase` is named correctly (as I have noted in the updated docs). If it is `True`, minor ticks will *not* be shown, which is exactly the opposite of what the name of the method implies. I did not find any explicit calls to this method in `matplotlib`. Am I free to modify the parameter name to say `labelMinor` and set `self.labelOnlyBase = not labelMinor`?